changelogs/changelog.yaml

```diff
@@ -88,3 +88,5 @@ releases:
     - fix_ci.yaml
     - restructure.yaml
     release_date: '2025-03-27'
+  6.0.0:
+    release_date: '2025-08-19'
```

Comment:
add a major release note and release summary explaining why this change is required, and revert old changes or check with pre-commit locally